etcdserver/cluster.go

```diff
@@ -66,6 +67,10 @@ type Cluster struct {
 	// TODO: upgrade it as last modified index
 	index uint64
 
+	// transport and members maintains the view of the cluster at index.
+	// This might be more upto date than what stores in the store since
+	// the index may be bigger than store index.
```

Comment:
It is mentioned once in the comment for index, but it makes it more explicit to explain it here too.